package.json

```diff
@@ -114,7 +114,7 @@
     "in-publish": "^2.0.0",
     "istanbul-instrumenter-loader": "^2.0.0",
     "jsdom": "^9.11.0",
-    "lint-staged": "^7.0.0",
```

Comment:
🤔 this did not change the resolved version, so maybe not needed? :)

Reply:
Interesting. The reason why I updated it was that I had issues with running it with node 10. Should not impact the code in any way.